app/assets/javascripts/admin.js

```diff
@@ -1,3 +1,23 @@
-// Place all the behaviors and hooks related to the matching controller here.
-// All this logic will automatically be available in application.js.
-// You can use CoffeeScript in this file: http://coffeescript.org/
+$(function() {
+  var allGuests = $('.admin-guest');
+
+  function filterAttending(e) {
```

Comment:
Oh that's right.. I mean doesn't matter either way. If it's just gonna be a time suck trying to figure it out, we can just leave the admin stuff as is.. not important.